test/index.js

```diff
@@ -584,6 +585,18 @@ test('deep clone', function (t) {
 	t.end();
 });
 
+test('deep clone; buffers are duplicated', function (t) {
+	var buff = fs.readFileSync('./index.js');
```

Comment:
we should be able to create a buffer without the `fs` module - tests need to run in the browser.